### PR TITLE
[stable/kibana] bump kibana, set default config empty

### PR DESCRIPTION
**Pull request by kimxogus**
Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

#### What this PR does / why we need it:
#12199 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes only kibana's #12199 issue

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/kibana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: kibana
-version: 2.0.0
-appVersion: 6.6.1
+version: 3.0.0
```
Comment (monotek) [inline stable/kibana/Chart.yaml]: is this a breaking change for people coming from <= 2.0? 
if not and its only a new feature chart version 2.1.0 would be better.

File: stable/kibana/values.yaml [hunk 2]
```diff
@@ -17,11 +17,11 @@ env: {}
   # SERVER_DEFAULTROUTE: "/app/kibana"
 
 files:
-  kibana.yml:
+  kibana.yml: {}
```
Comment (monotek) [inline stable/kibana/values.yaml:20]: I've just checked my kibana.yaml.

It looks like this:

```
elasticsearch.url: http://elasticsearch:9200
server.host: "0"
server.name: kibana
```

Looks totally ok to me, as its exactly that what was configured.

So where exactly comes this merge error from / how can i reproduce the error you're referring to in https://github.com/helm/charts/issues/12199 ?

Comment (kimxogus) [reply stable/kibana/values.yaml:20]: setting `elasticsearch.hosts` will produce kibana config error and setting `elasticsearch.url` as `null` or empty string `""` doesn't help.

As default config in this chart is exactly same with [kibana-docker's default config](https://github.com/elastic/kibana-docker/blob/master/.tedi/template/kibana.yml.j2), this default config won't do anything but making this deprecated config error

Comment (kimxogus) [reply stable/kibana/values.yaml:20]: #12378

File: stable/kibana/Chart.yaml (hunk 1 above)
Comment (kimxogus) [reply stable/kibana/Chart.yaml]: set `2.1.0`

**Comment by kimxogus** [comment]
fixed e2e test

**Pull request merged**